lib/helpers.js

```diff
@@ -57,4 +57,4 @@ function showError(givenMessage) {
     detail: detail,
     dismissable: true
   });
-}
+}
```

Comment:
Until @steelbrain can get a plugin put together for `ucompiler` I think I'm just going to stop fighting this missing newline in the transpiled code :confused:...